commit ca78c86645cfdc440588413c02bb0eeae8b402d2
Author: Vineetha Hari Pai <vineetha.hari.pai@canonical.com>
Date:   Fri Feb 16 12:27:14 2018 -0500

From: Vineetha Hari Pai<vineetha.hari.pai@canonical.com>
Decription: Disable libgcrypt reading /proc/sys/crypto/fips_enabled
file and going into FIPS mode. libgcrypt is not a FIPS
certified library.
Bug-Ubuntu: http://bugs.launchpad.net/bugs/1748310
Forwarded: not-needed

diff --git a/src/fips.c b/src/fips.c
index 3ab33f9..4f1496a 100644
--- a/src/fips.c
+++ b/src/fips.c
@@ -134,6 +134,10 @@ _gcry_initialize_fips_mode (int force)
       goto leave;
     }
 
+  /* Disabling reading fips_enabled file here to prevent
+     the library going automatically into FIPS mode.
+     LP: #1748310 */
+#if 0
   /* Checking based on /proc file properties.  */
   {
     static const char procfname[] = "/proc/sys/crypto/fips_enabled";
@@ -170,6 +174,7 @@ _gcry_initialize_fips_mode (int force)
         abort ();
       }
   }
+#endif
 
   /* Fips not not requested, set flag.  */
   no_fips_mode_required = 1;
